src/site/content/en/blog/schemeful-samesite/index.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Schemeful Same-Site
+subhead: The definition of "same-site" is evolving to include the URL scheme.
+  That means links between HTTP and HTTPS versions of a site now count as
+  cross-site requests. Upgrade to HTTPS by default to avoid issues where
+  possible or read on for details of what SameSite attribute values are needed.
+authors:
+  - bingler
+  - rowan_m
+date: 2020-11-18
+updated: 2020-11-18
+hero: schemeful-samesite-hero.jpg
+thumbnail: schemeful-samesite-thumbnail.jpg
+alt: Two separate plates of cookies. The plates represent the different
+  schemes, HTTP and HTTPS. The cookies represent cookies.
+tags:
+  - blog
+  - privacy
+  - security
+feedback:
+  - api
+---
+
+{% Aside %}
+This article is part of a series on the `SameSite` cookie attribute changes:
+- [SameSite cookies explained](/samesite-cookies-explained/)
+- [SameSite cookies recipes](/samesite-cookie-recipes/)
+- [Schemeful SameSite](/schemeful-samesite)
```

Comment:
There's a spelling discrepancy here, "Schemeful SameSite" is linking to "Schemeful Same-Site". Either is fine as long it's consistent. I suggest lower case "same-site" for the concept and using "SameSite" only for the cookie.

Reply:
Good catch.

"Schemeful Same-Site" is the name of this change. The concept of being the same site is "same-site" and the attribute that developers add to their cookies is "SameSite".

It's all very confusing and pedantic but that's all the more reason to stay consistent with it.

Reply:
Okay, if I understand correctly, the title is referring to "same-site" as the concept but uses title case? web.dev uses sentence case for titles, so it should be "Schemeful same-site" here but we can't be pedantic on both sides :)

Reply:
It is definitely a bit confusing. I've tried to ensure I'm using:

- `SameSite` when referring to the attribute
- "same-site" in prose

I would like to keep the URL as `schemeful-samesite` for consistency with the other posts. It looks like we're considering "Schemeful Same-Site" as a Proper Name though, meaning that it should be capitalised when referred to together but appears as "same-site" if we're referring just to the overall concept.

Reply:
I missed this until now, sorry about that.

The feature is named "Schemeful Same-Site" which is a proper noun and should have the capitalization if possible.

> I would like to keep the URL as schemeful-samesite for consistency with the other posts
What do you mean by consistent?